glustercli/cmd/georep.go

```diff
@@ -45,18 +46,14 @@ Rerun the Session Create command with --force once the issues related to Remote
 )
 
 var (
-	flagGeorepCmdForce            bool
-	flagGeorepShowAllConfig       bool
-	flagGeorepRemoteGlusterdHTTPS bool
-	flagGeorepRemoteGlusterdHost  string
-	flagGeorepRemoteGlusterdPort  int
+	flagGeorepCmdForce        bool
+	flagGeorepShowAllConfig   bool
+	flagGeorepRemoteEndpoints string
 )
 
 func init() {
 	// Geo-rep Create
-	georepCreateCmd.Flags().BoolVarP(&flagGeorepRemoteGlusterdHTTPS, "remote-glusterd-https", "", false, "Remote Glusterd HTTPS")
-	georepCreateCmd.Flags().StringVarP(&flagGeorepRemoteGlusterdHost, "remote-glusterd-host", "", "", "Remote Glusterd Host")
-	georepCreateCmd.Flags().IntVarP(&flagGeorepRemoteGlusterdPort, "remote-glusterd-port", "", 24007, "Remote Glusterd Port")
+	RootCmd.PersistentFlags().StringVar(&flagGeorepRemoteEndpoints, "remote-endpoints", "http://127.0.0.1:24007", "remote glusterd2 endpoints")
```

Comment:
This is specific to Geo-rep Create command for now. No other Geo-rep commands need this with the current design. We can change this to `georepCreateCmd.Flags().` to avoid showing up in other commands help output.

Reply:
my bad, forgot it add it to georepCreateCmd.Flags() . Made changes